test/MerkleLib.Fixture.ts

```diff
@@ -0,0 +1,17 @@
+import { getContractFactory } from "./utils";
+import hre from "hardhat";
+
+export const merkleLibFixture = hre.deployments.createFixture(async ({ deployments }) => {
```

Comment:
I wasn't sure if this was the type of fixtures we were wanting to use, but this is the type I'm familiar with (where hardhat keeps a snapshot of the fixture so it doesn't have to redeploy).

Reply:
ye, so this is why you needed the `hardhat-deploy` package but @nicholaspai and my "fixtures" thus far have not. we simply created functions that did the deployment, which is not great. we should converge on one standard that does the same thing throughout the repo.

I think what you have here is better, though, as it will actually do the snapshotting and no re-deployment. I can refactor the other "fixtures" once this PR is in.

Reply:
Sounds good. I think that's the right path to avoid this PR growing too large. I'm also happy to make this change as a follow-up.